OpenRA.Mods.Common/Traits/Buildings/RallyPoint.cs

```diff
@@ -149,14 +152,18 @@ public bool CanTarget(Actor self, in Target target, List<Actor> othersAtTarget,
 				var location = self.World.Map.CellContaining(target.CenterPosition);
 				if (self.World.Map.Contains(location))
 				{
-					cursor = this.cursor;
+					cursor = info.Cursor;
 
-					// Notify force-set 'RallyPoint' order watchers with Ctrl and only if this is the only building of its type selected
-					if (modifiers.HasModifier(TargetModifiers.ForceAttack))
+					// Notify force-set 'RallyPoint' order watchers with Ctrl
+					if (modifiers.HasModifier(TargetModifiers.ForceAttack) && !string.IsNullOrEmpty(info.ForceSetType))
```

Comment:
This means force setting doesn't work when there is no type defined, right? We should either mention that in the `Desc` or fall back to the old mechanism if no type is present.

Reply:
It is mentioned in the Desc!

The previous behaviour wasn't documented anywhere afaik (and didn't work very well anyway), so IMO its reasonable to ignore that and do nothing if it isn't set.

Reply:
`Used to group equivalent actors when force-setting a rallypoint (e.g. for Primary production).` doesn't imply to me at all that nothing will be set if the value is left as `null`.

Reply:
Well it does to me 😛 Do you have a suggestion for alternative wording that is more clear?

Reply:
Perhaps `Used to group equivalent actors to allow force-setting a rallypoint (e.g. for Primary production).`?